composer.json

```diff
@@ -19,7 +19,9 @@
     "require-dev": {
         "phpunit/phpunit": "^9",
         "mockery/mockery": "^1.6",
-        "phpstan/phpstan": "^1"
+        "phpstan/phpstan": "^1",
+        "dealerdirect/phpcodesniffer-composer-installer": "^1.0",
```

Comment:
Why do we need this package?

Reply:
To automatically setup phpcs for various IDEs - so our phpcs.xml config works OOB

Reply:
I tried to figure out what this package does but still don't get it.

> This composer installer plugin allows for easy installation of PHP_CodeSniffer
> No more symbolic linking of directories, checking out repositories on specific locations or changing the phpcs configuration.

Why do I need an installation package for "PHP_CodeSniffer" when you can install the tool without any other dependencies? `phpcs.xml` is read in automatically and/or can be integrated via the GitHub testing workflow.

Don't get me wrong. I don't want to criticize here, but always try to keep package dependencies as low as possible.

I think the IDE setup belongs on the machine of the developer. In my opinion, we can do without this dependency in this case.

Reply:
OK, I'll remove it.